### PR TITLE
ODP initial support v4

### PR DESCRIPTION
**Pull request by muvarov**

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1059,6 +1059,43 @@
...
+    AC_CHECK_HEADERS([odp_api.h])
+
+    LIBPODP=""
+    AC_CHECK_LIB(odp-linux, odp_init_global,, LIBODP="no")
+    if test "$LIBODP" = "no"; then
+        echo
+        echo "   WARNING!  Optional libodp library not found, go get it"
+        echo "   from http://www.opendataplane.org or your distribution:"
+        echo
+        echo " sudo apt-add-repository http://deb.opendataplane.org"
```
Comment (plaes) [inline configure.ac]: Do you really have to include the debian-specific instructions here?

File: doc/INSTALL.ODP [hunk 2]
```diff
@@ -0,0 +1,35 @@
+    OpenDataPlane (ODP) support
+    http://opendataplane.org
+
+-=Background
+    ODP is cross platform API describing networking operations on
+different SoC or NIC. Using ODP for Suricata allows to operate with all
+existence inputs plus SoC optimized implementations.
```
Comment (plaes) [inline doc/INSTALL.ODP]: s/existence/existing?

Comment (muvarov) [inline doc/INSTALL.ODP]: ok, will fix, thanks.

**Comment by muvarov** [comment]
corrected comments above and rebased to the latest master.

**Comment by muvarov** [comment]
Victor, I see that you added tags. Can I help somehow with future progress?

**Comment by inliniac** [comment]
Yeah, documentation of the YAML options in Sphinx (rst) format would be a great start.

A feature ticket in our redmine for ODP, with a link in the PR to the ticket.

'needprscript' means that someone of the team will need to run it through our 'prscript' QA step.

**Comment by inliniac** [comment]
I hope you'll be interested in reworking this for inclusion. Closing for now.

**Pull request closed without merging**